### PR TITLE
Order Book support

### PR DESCRIPTION
**Pull request by Nikita-Polyakov**

### CODE REVIEW

File: src/mappings/calls/orderBook.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  await updateHistoryElementStats(extrinsic, historyElement);
+}
+
+export async function orderBookCancelLimitOrderHandler(extrinsic: SubstrateExtrinsic): Promise<void> {
+  logStartProcessingCall(extrinsic);
+
+  const historyElement = await createHistoryElement(extrinsic)
+  const cancelEvents = extrinsic.events.filter(e =>
+    e.event.section === 'orderBook' &&
+    e.event.method === 'LimitOrderCanceled'
```
Comment (RustemYuzlibaev) [inline src/mappings/calls/orderBook.ts:47]: Does  method have the same name for batch and single?

Comment (Nikita-Polyakov) [reply src/mappings/calls/orderBook.ts:47]: Yes, when order is canceled, backend generates the `orderBook.LimitOrderCancelled` event, no difference which extrinsic was called

File: schema.graphql [hunk 2]
```diff
@@ -216,3 +216,77 @@ type StakingEra @entity {
...
+  dexId: Int!
+  baseAsset: Asset! @index
+  quoteAsset: Asset! @index
+  price: String!
+  priceChangeDay: Float
+  volumeDayUSD: Float
+  limitOrders: [OrderBookLimitOrder] @derivedFrom(field: "orderBook")
+  lastDeals: [OrderBookDeal!]!
+  updatedAtBlock: Int!
+  # status ?
```
Comment (RustemYuzlibaev) [inline schema.graphql]: status is indeed unnecessary for now

File: schema.graphql [hunk 3]
```diff
@@ -216,3 +216,77 @@ type StakingEra @entity {
...
+}
+
+type OrderBook @entity {
+  id: ID!
+  dexId: Int!
+  baseAsset: Asset! @index
+  quoteAsset: Asset! @index
+  price: String!
+  priceChangeDay: Float
+  volumeDayUSD: Float
```
Comment (RustemYuzlibaev) [inline schema.graphql]: will be calculated in USD?

**Review by RustemYuzlibaev — APPROVED** [review]

**Pull request merged**